junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/EnabledForJreRange.java

```diff
@@ -102,9 +104,36 @@
 	 * possible version.
 	 *
 	 * @see JRE
+	 * @see #maxFeatureVersion()
 	 */
 	JRE max() default JRE.OTHER;
 
+	/**
+	 * Java Runtime Environment feature version which should be used as the lower
```

Comment:
Linking "feature version" to `java.lang.Runtime.Version.feature()` would help to connect some dots. Perhaps you can't given your Java baseline?

Reply:
I was planning on documenting that more thoroughly, though I'm glad you made the connection.

We're currently invoking the deprecated `major()` method defensively via reflection for support on Java 9+, and I picked the "feature" terminology because of the new `feature()` method in Java 10+.

So I plan to explain that to some extent.

As for linking to the newer API, I think that actually might not be an issue, because I believe we are generating Javadoc with a JDK version later than JDK 8 (perhaps JDK 11 or later -- I'll have to check).